src/containers/shared/components/Account.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 export interface AccountProps {
-  link?: boolean
   account: string
+  link?: boolean
+  tag?: number
 }
 
 export const Account = (props: AccountProps) => {
-  const { link = true, account } = props
+  const { account, link = true, tag } = props
   const parts = account.split(':')
+  const computedTag = tag || parts[1]
```

Comment:
If there is no tag, does this cause an out of bounds exception?

Reply:
It does not.  There are tests that cover that.  It will just be undefined.